Doc/tutorial/venv.rst

```diff
@@ -149,14 +149,14 @@ install --upgrade`` to upgrade the package to the latest version:
         Successfully uninstalled requests-2.6.0
   Successfully installed requests-2.7.0
 
-``pip uninstall`` followed by one or more package names will remove the
-packages from the virtual environment.
+``python -m pip uninstall`` followed by one or more package names will
+remove the packages from the virtual environment.
 
-``pip show`` will display information about a particular package:
+``python -m pip show`` will display information about a particular package:
 
 .. code-block:: bash
 
-  (tutorial-env) $ pip show requests
+  (tutorial-env) $ python -m pip show requests
```

Comment:
The examples already in a virtual environment should not be changed because the whole point of the environment is to make sure the correct python and pip are being targeted.

Reply:
I can revert those if that's what it takes to get this merged (please confirm), but would like to point out first that
- they will then be the only places referring to the `pip` executable rather than `python -m pip` which might be a surprise to some readers, it gets more rare and there's no explanation for it, and
- the point of using `python -m pip` instead of `pip` is just as valid in virtual envs as it is outside of them